### ISSUE TITLE
Operator =in= with one argument

### ISSUE THREAD

**Issue by mifmax** [perplexhub/rsql-jpa-specification#3]
When I execute 
`localhost:8080/some_path/v1?rsql=id=in=(1)&limit=135&offset=0`, I have error 
`Unknown operator: =in=; nested exception is java.lang.IllegalArgumentException: Unknown operator: =in=`
But it's don't true. Operator =in= is known)

**Comment by perplexhub** [comment]
Hi @mifmax, could you try to use localhost:8080/some_path/v1?rsql=id=in=(1,1)&limit=135&offset=0 ? It helps to narrow down the problem. Thanks.

**Comment by perplexhub** [comment]
Thanks for your contribution. @mifmax

**Comment by perplexhub** [comment]
<dependency>
  <groupId>io.github.perplexhub</groupId>
  <artifactId>rsql-jpa-specification</artifactId>
  <version>3.0.1</version>
</dependency>

**Comment by perplexhub** [comment]
I modify the code as follow to mitigate the unexpected behavior when there are ^,* characters in the first argument. Changes are merged to version 3.0.2.

```java
			if (op.equals(IN)) {
				return builder.equal(attrPath, argument);
			}
			if (op.equals(NOT_IN)) {
				return builder.notEqual(attrPath, argument);
			}
```
